TTYC.Domain/Address.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace TTYC.Domain
+{
+    public class Address
+    {
+        public Guid Id { get; set; }
+        public string Street { get; set; }
+        public int HouseNumber { get; set; }
+        public int FlatNumber { get; set; }
+        public int Floor { get; set; }
+
+        public Guid UserId { get; set; }
```

Comment:
ProfileId? UserProfileId? Not clear what this Id is for

Reply:
But we dont have a User refence. And why is it specially grouped with UserProfile?